task/clamav-scan/0.2/clamav-scan.yaml

```diff
@@ -0,0 +1,216 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "virus, konflux"
+  name: clamav-scan
+spec:
+  description: >-
+    Scans the content of container images for viruses, malware, and other malicious content using ClamAV antivirus scanner.
+  results:
+    - name: TEST_OUTPUT
+      description: Tekton task test output.
+    - name: IMAGES_PROCESSED
+      description: Images processed in the task.
+  params:
+    - name: image-digest
+      description: Image digest to scan.
+    - name: image-url
+      description: Image URL.
+    - name: docker-auth
+      description: unused
+      default: ""
+    - name: ca-trust-config-map-name
+      type: string
+      description: The name of the ConfigMap to read CA bundle data from.
+      default: trusted-ca
+    - name: ca-trust-config-map-key
+      type: string
+      description: The name of the key in the ConfigMap that contains the CA bundle data.
+      default: ca-bundle.crt
+
+  steps:
+    - name: extract-and-scan-image
+      image: quay.io/konflux-ci/clamav-db:latest
```

Comment:
Can we just add comment below about that this floating tag is being build daily, hence the interpretation.